recipes/xdem/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = 'xdem' %}
+{% set version = '0.0.3' %}
+
+package:
+  name: '{{ name | lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/GlacioHack/xdem/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 06aa480c8ac749248ef5b2fc93c95ebabea344d4930251a27a44565f90b5f4c8 
+
+build:
+  number: 0
+  noarch: python
+  script: '{{ PYTHON }} -m pip install . -vv'
+
+requirements:
+  host:
+  - python >=3.7
+  - pip
+
+  run:
+  - python >=3.7
+  - proj >=7.2
+  - geopandas
+  - numba
+  - matplotlib-base
+  - opencv
+  - pyproj
+  - rasterio
+  - scipy
+  - tqdm
+  - scikit-image
+  - scikit-gstat
+  - pytransform3d
+  - geoutils
+
+test:
+  imports:
+  - xdem
+
+about:
+  home: https://github.com/GlacioHack/xdem
+  license: MIT
+  license_file: LICENSE
+  summary: Set of tools to manipulate Digital Elevation Models (DEMs)
+  doc_url: https://xdem.readthedocs.io
+  dev_url: https://github.com/GlacioHack/xdem
+
+extra:
+  recipe-maintainers:
+  - erikmannerfelt
+  - adehecq
+  - rhugonnet
```

Comment:
@adehecq and @rhugonnet are you OK becoming maintainers here?

Reply:
:+1:

Reply:
Yes